### ISSUE TITLE
Google Colab

### ISSUE THREAD

**Issue by erigamonte** [selimsef/dfdc_deepfake_challenge#26]
Hello! I need to ask, Is it possible to run this code on Google Colab?

**Comment by Car0liViA** [comment]
I want to ask too, any one could help?

**Comment by AmerYassir** [comment]
same

**Comment by yaylymov** [comment]
> 

Yes it's totally possible. Mount your google drive to google colab and clone the repo there, so that you don't have to clone the repo every time you log out of google colab

**Comment by AyeshaMalikAyesha** [comment]
@yaylymov Kindly provide guidance on the utilization of the pre-trained model obtained through the cloning process, as instructed earlier. Following the cloning procedure, we seek assistance in conducting tests using this pre-trained model on our designated test set. We kindly request a detailed outline of the steps involved in performing this testing process.
